automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -117,22 +119,53 @@ function failure_check() {
 
     echo
     echo "COPY ARTIFACTS"
+    echo "RESULTS"
+    echo
     echo
     cp -r ./results /builds/worker/artifacts
+    ls -la "${RESULTS_DIR}" 
 
     echo
+    echo
+    echo "RESULTS"
     echo "RESULTS"
     echo
+    echo
     ls -la ./results
+
     echo
+    mkdir -p /builds/worker/artifacts/github
     echo
+    chmod +x $PATH_TEST/parse-ui-test.py
+    $PATH_TEST/parse-ui-test.py \
+        --exit-code "${exitcode}" \
+        --log flank.log \
+        --results "${RESULTS_DIR}" \
+        --output-md "${ARTIFACT_DIR}/github/customCheckRunText.md" \
+	--device-type "${device_type}"
 }
 
+echo
+echo "FLANK VERSION"
+echo
+$JAVA_BIN -jar $FLANK_BIN --version
+echo
+echo
+
 echo
 echo "EXECUTE TEST(S)"
```

Comment:
🤔 I'm wondering why `parse-ui-test.py` is called before the test execution. What is the rationale?

Reply:
we were looking at various solutions for outputting test results into the dev workflow.  Tom and Owlish wanted to work on a solution that would work on multiple projects and he proposed this file which we leverage also on fenix (and will eventually port over to R-B as well).  We could roll it up into ui-test, but if we ever run other (i.e. Unit) tests on Firebase, we could leverage this same, separate parsing file to parse the test results out. I'm open to suggestions if you think it might be implemented differently

Reply:
Okay, I don't have any opinions, I was just wondering 🙂 Let's keep it this way!